examples/tpu/tpuvm_multinode.yml

```diff
@@ -0,0 +1,12 @@
+name: tpuvm_multinode
+
+resources:
+  accelerators: tpu-v2-8
+  accelerator_args:
+    runtime_version: tpu-vm-base
+    tpu_vm: True
+
+num_nodes: 2
+
+run: |
+  sleep 120
```

Comment:
Why don't we use a more informative run command like `echo My hostname: $(hostname)`?

Reply:
In addition to this, can we have a DL training example that uses multi-node TPU VMs?

Reply:
1) Good point. I added the echo command before sleep. The reason I use `sleep` is to test whether `sky cancel` and our job queue work or not in the smoke test.
https://github.com/skypilot-org/skypilot/pull/1185/files#diff-da9ef240243bf38ab4e33d1f8a2a64caee7677dc780e58557f3c676687a4b334R424

2) I feel like a DL training example won't give us more testing coverage than `sleep`. because the training code will only be run on a single TPU VMs. Or you think otherwise?

Reply:
Sounds very good!